components/monitoring/base/kustomization.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - install_prometheus.yaml
+  - prometheus-view.yaml
+
```

```
error validating data: ValidationError(OperatorGroup.spec.targetNamespaces): invalid type for com.coreos.operators.v1.OperatorGroup.spec.targetNamespaces: got "string", expected "array"
```
`one or more objects failed to apply (dry run)`
```suggestion
commonAnnotations:
  argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

---

@Michkov I am curious, how are you testing this? 
 `$kubectl kustomize .` generates the yamls properly here. any ideas on how should I test them?

---

having dev cluster, using preview mode to deploy on it. In this case it should be possible to reproduce by creation of argoCD app.